src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -764,6 +774,10 @@ inline const char *InstructionSetToString(CORINFO_InstructionSet instructionSet)
             return "VectorT128";
         case InstructionSet_Rcpc2 :
             return "Rcpc2";
+        case InstructionSet_Sve :
```

Comment:
Curious, why is there two instruction sets here? For the other ones it was because of AArch32, but there's no AArch32 SVE.

Reply:
AIUI, we need both to matching the existing structure. All the functions will be in Sve and Sve.Arm64 will be empty.